docker/Dockerfile.ci_gpu

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 # CI docker GPU env
-# tag: v0.56
+# tag: v0.57
```

Comment:
we want to avoid updating the tag before build and test the correctness. I will send a separate PR to do so

Reply:
got it, i'll revert the changes.